### PR TITLE
Minor Fixes and Updates

### PR DESCRIPTION
**Pull request by Jacques00**
* Some minor bug and typo fixes.
* Update stats.
* Aliss hardlight strap-on fixes.
* New Item/Character: Siegwulfe, written by Savin.
 * Unlocked at Level 5 or higher.
 * Purchased from Nerrasa (normal version) or Dr. Badger (bimbo version).
 * Can only be bought one at a time--can be sold and bought again, if necessary.
 * Normal version does kinetic damage, Bimbo version does tease damage.
 * Can also be "interacted" with in the general inventory menu as well as crew menu. Sex included.
* Ramis can now sex with masculine futas and party with sexless characters.
* Added some specific check functions in the creature script that relate to genitals and femininity.
* New Adjatha busts for Ilaria.
* New Shou busts for security drone and Jerome.
* Add cumflation-related adjectives to the belly parser. 
* Adjusted Ilaria to support cumflation.

### CODE REVIEW

File: classes/Creature.as [hunk 1]
```diff
@@ -3140,9 +3140,39 @@
 		{
 			return hasPerk("Brute Speech");
 		}
+		public function isSexless():Boolean
+		{
+			if(!hasCock() && !hasVagina()) return true;
+			return false;
+		}
+		public function isMan():Boolean
```
Comment (Fenoxo) [inline classes/Creature.as]: You're confusing gender and sex. These should probably be isMale and isFemale.

Oh god, I've gone full tumblr.

File: classes/Characters/Siegwulfe.as [hunk 2]
```diff
@@ -115,7 +111,7 @@
 		{
 			hairColor = "dark blonde";
 			skinTone = "tanned";
-			
+			lipMod = 5;
```
Comment (Fenoxo) [inline classes/Characters/Siegwulfe.as]: lipMod 5 is gonna be fucknormous. Way bigger than how you drew her. If she's already got high feminity, 2-3 is probably as big as you wanna go.

File: classes/Creature.as (hunk 1 above)
Comment (Jacques00) [inline classes/Creature.as]: Heheh, I might have tossed the terms in a blender, yes. It was just what I was thinking at the time--but the function names should be fixed now.

Comment (Fenoxo) [inline classes/Creature.as]: I saw. Nice work!

**Pull request merged**